packages/cli/src/license/license.service.ts

```diff
@@ -110,22 +108,50 @@ export class LicenseService {
 		return this.license.getManagementJwt();
 	}
 
-	async activateLicense(activationKey: string) {
+	async activateLicense(activationKey: string, eulaUri?: string) {
 		try {
-			await this.license.activate(activationKey);
+			await this.license.activate(activationKey, eulaUri);
 		} catch (e) {
-			const message = this.mapErrorMessage(e as LicenseError, 'activate');
+			// Check if this is a EULA_REQUIRED error from license server
+			if (this.isEulaRequiredError(e)) {
+				const { LicenseEulaRequiredError } = await import(
```

Comment:
Is this inline import on purpose? Seems like to me that for a small error class we can import it on the top